src/main/java/mods/eln/transparentnode/autominer/AutoMinerSlowProcess.java

```diff
@@ -208,19 +208,12 @@ public void process(double time) {
     }
 
     private IInventory getDropInventory() {
-        TileEntityChest chestEntity = null;
-        {
-            Coordonate c = new Coordonate(2, -1, 0, miner.world());
+        IInventory chestEntity = null;
+        for (int x = 2; x >= 1; x--) {
+            Coordonate c = new Coordonate(x, -1, 0, miner.world());
             c.applyTransformation(miner.front, miner.coordonate());
-            if (c.getTileEntity() instanceof TileEntityChest) {
-                chestEntity = (TileEntityChest) c.getTileEntity();
-            }
-        }
-        {
-            Coordonate c = new Coordonate(1, -1, 0, miner.world());
-            c.applyTransformation(miner.front, miner.coordonate());
-            if (c.getTileEntity() instanceof TileEntityChest) {
-                chestEntity = (TileEntityChest) c.getTileEntity();
+            if (c.getTileEntity() instanceof IInventory) {
+                chestEntity = (IInventory) c.getTileEntity();
```

Comment:
I would not call the variable chest anymore, but this is really a detail and does not matter.